pages/osx/sed.md

```diff
@@ -10,9 +10,9 @@
 
 `sed -E 's/{{regex}}/{{replace}}/g' {{filename}}`
 
-- Replace all occurrences of a string in a file, overwriting the file (i.e. in-place):
+- Replace all occurrences of a string [i]n a file, overwriting the file (i.e. in-place):
 
-`sed --in-place='' 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed -i '' 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
We generally prefer long-form options in tldr pages for clarity.

Reply:
See https://github.com/tldr-pages/tldr/pull/4781#discussion_r510128984

Reply:
@sbrl Please check https://github.com/tldr-pages/tldr/pull/4781#discussion_r510128984

Reply:
Oops! I must have missed this. Ignore this suggestion.